lib/sources.nix

```diff
@@ -17,6 +17,8 @@ rec {
     (type == "directory" && (baseName == ".git" || baseName == ".svn" || baseName == "CVS" || baseName == ".hg")) ||
     # Filter out backup files.
     lib.hasSuffix "~" baseName ||
+    builtins.match "^.*\.sw[a-z]$" baseName != null ||
```

Comment:
This matches files like `foo.swp` where the old one didn't. Probably not a big deal though.

Maybe 
````
builtins.match "^\.sw[a-z]$" baseName != null ||
builtins.match "^\..*\.sw[a-z]$" baseName != null ||
````
could work also.